File: app/src/main/java/com/example/rick_and_morty_android_networking_example_app/domain/models/CharacterResponse.kt
package com.example.rick_and_morty_android_networking_example_app.domain.models

import com.example.rick_and_morty_android_networking_example_app.data.api.dto.CharacterDTO
import com.google.gson.annotations.SerializedName


data class Info(
    val count: Int?,
    val next: String?,
    val pages: Int?,
    val prev: String?
)

data class CharacterResponse(
    val info: Info,
    val results: List<CharacterDTO>
)

File: Rick & Morty MasterDetail App/Rick & Morty MasterDetail App/Domain/Models/CharacterResponse.swift
//
//  CharacterResponse.swift
//  Rick & Morty MasterDetail App
//
//  Created by Miguel on 5/18/23.
//

import Foundation

struct Info: Codable {
    let count: Int?
    let pages: Int?
    let next: String?
    let prev: String?
}

struct CharacterResponse: Codable {
    let info: Info?
    let results: [CharacterDTO]?
}
